README.md

```diff
@@ -1 +1,179 @@
+- [📖 \[SpaceX-Project\] ](#-SpaceX-Project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Build](#build)
+    - [Run](#run)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [© Acknowledge ](#-acknowledge-)
+  - [⭐ Show your support ](#-show-your-support-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+# 📖 [SpaceX-project] <a name="about-project"></a>
+
+*[SpaceX-project]* a WebApp that shows three pages: rockets, missions and My profile. here, users can reserve rockets and join spacial missions with spaceX company.
+<div align="center">
+
+  <img src="./src/images/Screenshotmissions.png" alt="Spacex-project2" width="750"  height="auto" />
+  <img src="./src/images/rockets.png" alt="Spacex-project" width="750"  height="auto" />
+
+  <br/>
+
+</div>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML" target="_blank" rel="noopener noreferrer">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS" target="_blank" rel="noopener noreferrer">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript" target="_blank" rel="noopener noreferrer">Javascript</a></li>
+    <li><a href="https://webpack.js.org/" target="_blank" rel="noopener noreferrer">WebPack</a></li>
+    <li><a href="https://es.reactjs.org/" target="_blank" rel="noopener noreferrer">React</a></li>
+    <li><a href="https://es.redux.js.org/" target="_blank" rel="noopener noreferrer">Redux</li>
+  </ul>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- *Dinamically displays the pages that were build by using react function based components.*
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="https://books-store-8hzd.onrender.com" target="_blank" rel="noopener noreferrer">SpaceX-project - Live Demo</a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need `git` locally installed. Please got to [Getting Started - Installing Git guide](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and follow the steps described for your system to install `git`.
+
+### Setup
+
+Clone this repository to your desired folder:
+```sh
+  cd my-folder
+  git clone git@github.com:juanpa8830/spacex-project.git
+```
+
+### Install
+
+Install this project with:
+```sh
+  cd Books-store
```

Comment:
- [ ] Please consider changing the `Books-store` to the correct directory name! `spacex-project` 🤓 🔧